lib/qa/authorities/persons.rb

```diff
@@ -30,7 +30,10 @@ def persons
         end
 
         def creators
-          @creators ||= Actor.where('surname ILIKE :q OR given_name ILIKE :q OR psu_id ILIKE :q', q: "%#{term}%")
+          @creators ||= begin
+                          Actor.where('surname ILIKE :q OR given_name ILIKE :q OR psu_id ILIKE :q', q: "%#{term}%") |
+                            Actor.where(psu_id: identities.map(&:user_id))
```

Comment:
This seems expensive to do with two queries + ruby. Can we not do this all in a single SQL query? 

```ruby
@creators ||= Actor
  .where('surname ILIKE :q OR given_name ILIKE :q OR psu_id ILIKE :q', q: "%#{term}%")
  .or(Actor.where(psu_id: identities.map(&:user_id)))
```